src/nft/queries/openSea/OSCollectionsFetcher.ts

```diff
@@ -13,17 +13,12 @@ export const OSCollectionsFetcher = async ({ params }: any): Promise<WalletColle
   const r = await fetch(`https://api.opensea.io/api/v1/collections?${new URLSearchParams(params).toString()}`)
   const walletCollections = await r.json()
   if (walletCollections) {
-    return walletCollections
-      .filter(
-        (collection: any) =>
-          collection.primary_asset_contracts.length && collection.primary_asset_contracts[0].schema_name === 'ERC721'
```

Comment:
should we keep the filter on collection.primary_asset_contracts.length so we make sure its not undefined?

Reply:
Great catch, we do still want this